src/manifest/mod.ts

```diff
@@ -93,29 +99,27 @@ export class SlackManifest {
     }
 
     if (def.features?.appHome) {
-      const { messagesTabEnabled, messagesTabReadOnlyEnabled } =
-        def.features.appHome;
-
-      if (messagesTabEnabled !== undefined) {
-        manifest.features.app_home.messages_tab_enabled = messagesTabEnabled;
-      }
-      if (messagesTabReadOnlyEnabled !== undefined) {
+      const {
+        home_tab_enabled,
+        messages_tab_enabled,
+        messages_tab_read_only_enabled,
+      } = def.features.appHome;
```

Comment:
@shapirone. Yeah good catch. These are snake case now because the underlying `ManifestAppHomeSchema` is in snake_case (in following with how ManifestSchema) is serialized. I think types in `manifest_schema.ts` types should match the output, so I'm not super inclined to change this back. 

Folks who do have this as camelCase currently would need to correct the type now as part of these changes if they have it in their manifest, though I don't think that should be too big of a problem for devs. Maybe we can add a release note? 

The bigger vision to have devs work in camelCase and conversion to snake_case happening under the hood I agree with completely though. To that end, I noticed camelCase vs. snake_case is used interchangeably in our reverseString sample, so it would be good to revisit that holistically in a separate PR. Depending on how deeply we decide to make casing consistent, it could raise type errors in existing apps (say in a DefineFunction),

Reply:
Added a `CamelCased` conversion to resolve this issue and synced about this offline!